spack.yaml

```diff
@@ -1,8 +1,9 @@
 spack:
   specs:
+    - hpx build_type=Release cxxstd=14 networking=none
+    - boost cxxstd=14 ~atomic~math~iostreams~locale~graph~random~serialization~signals~timer~test~wave~mpi~python
```

Comment:
This dependency is maybe not enough on all systems. 

See https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/HPX_SetupBoost.cmake

From it, I was able to figure out that components
- `context`
- `thread`
- `chrono`

are required on `APPLE`, `BLUEGENEQ`, and `ARM`.

I don't think that adding them by default my affect HPX decision about which `coroutine` to use, but I would check it (or ask some HPX guy) because it may affect performances.